extensions/src/main/java/org/pentaho/platform/web/http/api/resources/WorkerNodeActionInvokerAuditor.java

```diff
@@ -70,4 +70,9 @@ protected void makeAuditRecord( final float time, final String messageType,
   private String getValue( Map<String, Serializable> actionParams, String key ) {
     return actionParams.get( key ) != null ? actionParams.get( key ).toString() : "";
   }
+
+  @Override
+  public boolean isSupportedAction( IAction action ) {
+    return false; // not needed
```

Comment:
Since the auditor is a decorator, that adds some capability to decorate an underlying invoker,  for "cleanliness" purposes, it should probably delegate to the action invoker it decorates.